apps/server/src/modules/system/console/ldap-sync.console.ts

```diff
@@ -0,0 +1,29 @@
+import { FeathersServiceProvider } from '@shared/infra/feathers/feathers-service.provider';
+import { Logger } from '@src/core/logger';
+import { Command, Console } from 'nestjs-console';
+
+@Console({ command: 'ldap-sync', description: 'ldap sync console' })
+export class LdapSyncConsole {
+	constructor(private readonly feathersServiceProvider: FeathersServiceProvider, private logger: Logger) {}
+
+	@Command({
+		command: 'force-full-sync',
+		description: 'runs a full ldap sync',
+	})
+	async fullLdapSync(): Promise<void> {
+		const ldapServiceData = {
+			target: 'ldap',
+			forceFullSync: true,
+		};
+		try {
+			const service = this.feathersServiceProvider.getService('/sync');
```

Comment:
does this really work? since its not running in a web environment, is the feathers stuff even there?